Include/Rover/TrialView.hpp

```diff
@@ -16,7 +16,7 @@ namespace Rover {
       using Sample = S;
 
     private:
-      using GetPtr = std::function<const Sample* (std::size_t)>;
+      using Get = std::function<const Sample& (std::size_t)>;
```

Comment:
Put the private section at the bottom of the class.

Reply:
Not sure how to handle this then. The alias is privately used by `ConstIterator`, which is a public nested class. Should I make the alias public then?

Reply:
The private section goes after `ConstIterator`, so it won't otherwise see the alias.

Reply:
Okay, leave it as is in that case.